test cases/vala/28 generated ui file/meson.build

```diff
@@ -0,0 +1,34 @@
+project('demo', 'c', 'vala')
+
+gnome = import('gnome')
+
+deps = [
+  dependency('glib-2.0', version : '>=2.50'),
+  dependency('gobject-2.0'),
+  dependency('gtk+-3.0'),
+]
+
+ui_tgt = custom_target(
+  input: 'TestBox.ui.in',
+  output:  'TestBox.ui',
+  command: [find_program('cat')],
+  feed: true,
+  capture: true,
+)
```

Comment:
use `fs.copyfile` for this

Reply:
`fs.copyfile` runs at configuration time, which means that it will always run before valac, thus never triggering the primary issue. Are you still sure it should be changed?

Reply:
Hmm, great point. Keep it as is. This is a nifty way to copy a file too. Didn't even realize Windows had cat.